packages/api/core/test/fast/require-search_spec.ts

```diff
@@ -13,4 +13,10 @@ describe('require-search', () => {
     const resolved = requireSearch(__dirname, ['../../src/util/forge-config']);
     expect(resolved).to.equal(findConfig);
   });
+
+  it('should throw if file exists but fails to load', () => {
+    expect(() => {
+      requireSearch(__dirname, ['../fixture/require-search/throw-error.js']);
```

Comment:
In keeping with the other tests, shouldn't the extension be omitted?

```suggestion
      requireSearch(__dirname, ['../fixture/require-search/throw-error']);
```